frontend/interfaces/software.ts

```diff
@@ -242,27 +242,34 @@ export interface ISoftwareInstallVersion {
   installed_paths: string[];
 }
 
+export interface IHostSoftwarePackage {
+  name: string;
+  self_service: boolean;
+  icon_url: string;
+  version: string;
+}
+
+export interface IHostAppStoreApp {
+  appStoreId: string;
```

Comment:
Can we use snake case here to match the casing being returned by the API?

Reply:
yah will do, dont know why i had it this way to begin with.